core/config/docs/core.toml

```diff
@@ -470,6 +470,12 @@ MaxEncryptedSecretsSize = '26.40kb' # Default
 # MaxConfigSize is the maximum size of a config that can be fetched from the given config url.
 MaxConfigSize = '50.00kb' # Default
 
+[Capabilities.WorkflowRegistry.Limits]
+# WorkflowsGlobalLimit is the maximum number of workflows that can be registered globally.
+WorkflowsGlobalLimit = 50 # Default
```

Comment:
Do we need to say `Limit` twice? Or could we drop either the extra wrapper or the suffix?